source/dashboards/dashboards.markdown

```diff
@@ -55,12 +59,26 @@ The **Areas** dashboard is prepopulated by default and shows your {% term entiti
 Screenshot of the Areas default dashboard.
 </p>
 
-After you [added the dashboard](#creating-a-new-dashboard), you can edit the **Areas** dashboard:
+#### Limitations of the Areas dashboard
 
-1. In the top-right corner, select the {% icon "mdi:pencil" %} icon.
-2. You can show or hide sections, rearrange the content, or [add badges](/dashboards/badges/).
+1. Not all devices are automatically added to the **Areas** dashboard {% icon "mdi:house" %} (the first view). Some motion detectors, door/window contact sensors, and BLE/third-party thermometer devices are not added to the {% icon "mdi:house" %} view, but only to the area-specific view.
+2. Certain entity types and helpers are omitted from the **Areas** dashboard. For example, `input_select` and `input_boolean` entities do not appear, even when assigned to an area.
```

Comment:
1. These sensors will not even appear in the area specific view. Only few sensors are displayed (temperature/humidity of the area and door, window, gate binary sensors)
2. `input_select` and `input_boolean` will be added in HA `2025.5` (https://github.com/home-assistant/frontend/pull/25073)